Orchard.sln

```diff
@@ -683,7 +683,7 @@ Global
 		{CA48C824-1B11-46FF-9D43-AA0BC6955345} = {F23AC6C2-DE44-4699-999D-3C478EF3D691}
 		{23B628D8-4DCF-4EE1-AEA2-8CFF6C8B80B0} = {90030E85-0C4F-456F-B879-443E8A3F220D}
 		{A1B729D1-2BB2-4206-869B-839BF35D0ED5} = {90030E85-0C4F-456F-B879-443E8A3F220D}
-		{C0BAFD34-B9A2-4DBD-8FCA-133CF697CC0F} = {275E087F-A4E2-4A7B-A924-ED68E3A52086}
```

Comment:
In oauth2 branch Orchard.SiteSettings.Core was out of the Core Solution folder. This fixes it.